ci/run-ci.py

```diff
@@ -0,0 +1,474 @@
+#!/usr/bin/env python3
+#
+# ABOUT
+#   This script runs the full substra CI pipeline.
+#   "Gotta test them all!" ™
+#
+#   This script performs the following operations:
+#     - Create a new GKE cluster
+#     - Clone the repos:
+#       - hlf-k8s
+#       - substa-backend
+#       - substra-tests
+#     - Build the docker images using Google Cloud Builds
+#     - Deploy these images using skaffold
+#     - Wait for the substra application stack to be up and ready (i.e. wait
+#       for the substra-backend servers readiness probe)
+#     - Run the full 'substra-tests' test suite
+#     - Destroy the cluster
+#
+# AUTOMATIC CLUSTER DESTRUCTION
+#   We ensure the GKE cluster always gets destroyed at the end of the execution
+#   of this script.
+#
+#   We use 2 mechanisms:
+#   - When the script exits, we destroy the cluster as the final step. That's
+#     the case even if the script exits with an error or with an interruption
+#     (Ctrl+C). See `trap` command.
+#   - In any other "abrupt shutdown" case (system shutdown, Travis build
+#     canceled, etc), we use a daily scheduled task to destroy stale clusters
+#     after 24h. See https://console.cloud.google.com/functions/details/us-central1/clean-substra-tests-ci-deployment
+#
+# USEFUL LINKS
+#   - Travis build history: https://travis-ci.org/github/SubstraFoundation/substra-tests/builds
+#   - Stale cluster deletion script:
+#       https://console.cloud.google.com/functions/details/us-central1/clean-substra-tests-ci-deployment
+import os
+import time
+import json
+import shutil
+import string
+import random
+import argparse
+import subprocess
+import sys
+
+CLUSTER_NAME_ALLOWED_PREFIX = 'substra-tests'
+CLUSTER_NAME = ''
+CLUSTER_MACHINE_TYPE = 'n1-standard-8'
+
+CLUSTER_VERSION = '1.15.11-gke.1'
+CLUSTER_PROJECT = 'substra-208412'
+CLUSTER_ZONE = 'europe-west4-a'
+
+SERVICE_ACCOUNT = 'substra-tests@substra-208412.iam.gserviceaccount.com'
+KEY_SERVICE_ACCOUNT = 'substra-208412-3be0df12d87a.json'
+
+SUBSTRA_TESTS_BRANCH = 'nightly-tests'
```

Comment:
@todo change this :-)

Reply:
It's here: https://github.com/SubstraFoundation/substra-tests/pull/98

It's not merged yet because we want to be able to keep testing changes in Travis if necessary. If we reference the master branch here directly  then we can't run the tests in travis because the master branch is still the old version.